insonmnia/hardware/hardware.go

```diff
@@ -65,6 +79,30 @@ func (h *Hardware) SetNetworkIncoming(IPs []string) {
 	}
 }
 
+func (h *Hardware) AskPlanResources() *sonm.AskPlanResources {
+	result := sonm.AskPlanResources{}
+	result.CPU.CorePercents = uint64(h.CPU.GetDevice().GetCores()) * 100
```

Comment:
Looks like nil pointer deref as CPU is pointer
https://play.golang.org/p/uXC5LrYhO7G